src/storage/src/hummock/store/version.rs

```diff
@@ -292,10 +367,22 @@ impl HummockReadVersion {
                     self.staging
                         .imm
                         .retain(|imm| imm.epoch() > max_committed_epoch);
+
+                    self.staging
+                        .merged_imm
+                        .retain(|merged_imm| merged_imm.epoch() > max_committed_epoch);
```

Comment:
Is it possible for a mergfed_imm to spand across max_committed_epoch? For exmaple, mce=4 and merged_imm has epoch [3,4,5].

Reply:
It should not happen. Because we only merge sealed epoch data. So a merged_imm would not cross the checkpoint epoch boundary.